src/analyze/analyze.ts

```diff
@@ -26,15 +26,15 @@ export async function analyze(
     urlResolver: new PackageUrlResolver(),
   });
 
-  const elements = new Set<Element>();
-  const isInDependency = /(\b|\/|\\)(bower_components|node_modules)(\/|\\)/;
+  const isInTests = /(\b|\/|\\)(test)(\/|\\)/;
```

Comment:
I think we need a TODO to address magic constant here.  Also, it feels like \b should cut it for slashes too, so maybe `\btest\b`?